sql/core/src/test/scala/org/apache/spark/sql/test/SQLTestUtils.scala

```diff
@@ -320,6 +320,17 @@ private[sql] trait SQLTestUtilsBase
     )
   }
 
+  /**
+   * Drops global view `viewName` after calling `f`.
+   */
+  protected def withGlobalView(viewNames: String*)(f: => Unit): Unit = {
```

Comment:
I think `withTempView("global_temp.view_name")` should work?

Reply:
If written like that, then in the code we have to write like this
<img width="736" alt="image" src="https://github.com/apache/spark/assets/15246973/d99021b5-ad00-450d-8ee2-7152c626c430">

<img width="896" alt="image" src="https://github.com/apache/spark/assets/15246973/ca816b66-24e1-49e4-b83e-9256810952cf">

https://github.com/apache/spark/blob/e499ba80be6828dcf0c34d3f06f1cd12321ce5b8/sql/core/src/main/scala/org/apache/spark/sql/execution/SparkSqlParser.scala#L550-L554